templates/components/common/body.html

```diff
@@ -1,8 +1,8 @@
-<div class="body" id='main-content' data-currency-code="{{currency_selector.active_currency_code}}">
+<main class="body" id='main-content' role='main' data-currency-code="{{currency_selector.active_currency_code}}">
```

Comment:
Do we need role='main' if we are using main tag?

Reply:
it's recommended on MDN to add it for supporting old browsers like IE11